spec/unit/puppet-strings/yard/handlers/ruby/provider_handler_spec.rb

```diff
@@ -27,6 +27,34 @@
     end
   end
 
+  describe 'parsing a provider with an invalid docstring assignment' do
+    let(:source) { <<-SOURCE
+Puppet::Type.type(:custom).provide :linux do
+  @doc = 123
```

Comment:
curiosity to me.. is this what you referenced for puppet 3x?  do the various types of docstrings have to be instance vars?